flow/designs/asap7/minimal/README.md

```diff
@@ -0,0 +1,118 @@
+Report on a design prior to setting up a configuration
+======================================================
```

Comment:
Just a small nit: I would prefer to keep consistency across all `.md` files and use `#` at the beginning of the line to define headers and subheaders.